napari/_qt/_qapp_model/qactions/_help.py

```diff
@@ -107,4 +112,11 @@ def _show_about(window: Window):
         callback=partial(open, url=HELP_URLS['homepage']),
         menus=[{'id': MenuId.MENUBAR_HELP, 'group': MenuGroup.NAVIGATION}],
     ),
+    Action(
+        id='napari.window.help.log',
```

Comment:
```suggestion
        id='napari.window.help.show_log',
```

very small nitpick, could we match the callback?